src/libs/KeyboardShortcut/index.js

```diff
@@ -124,6 +126,56 @@ function subscribe(key, callback, modifiers = 'shift', captureOnInputs = false)
     return () => unsubscribe(key);
 }
 
+
```

Comment:
there's an extra line of whitespace here.

Reply:
Oops